enum_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list.flatten
+  end
+
```

Comment:
- [ ] [OPTIONAL] This milestone should be created with peer programming in mind, make sure to pay attention to this in future projects